docs/Manual/Deployment/ArangoDBStarter/README.md

```diff
@@ -15,3 +15,18 @@ Deploying using the ArangoDB Starter
 
 - [_Starter_ using processes](../Cluster/UsingTheStarter.md)
 - [_Starter_ using Docker containers](../Cluster/UsingTheStarter.md#using-the-arangodb-starter-in-docker)
+
+Best Practices
+--------------
+
+### Linux user used to run _arangodb_
+
+It is strongly suggested to not run the _ArangoDB Starter_ process (_arangodb_)
+as _root_. We suggest creating a dedicated Linux user for this purpose.
+
+Assuming the user you want to use is called _arangodb_, to run the _Starter_,
+on Linux systems, you might use a command similar to the following:
+
+```
+sudo su arangodb -s /bin/bash -c "arangodb --all.server.uid=$(id -u arangodb) ... AnyStarterOptionHere"
+```
```

Comment:
I suggest to add a hint that for production use one should use something like a `systemd` service to run the starter. There one can configure the user under which it is run easily and this makes sure that things are restarted automatically, for example after a reboot. Details can be found here: https://www.arangodb.com/2019/01/run-multiple-versions-arangodb/